src/lib/booleans.js

```diff
@@ -0,0 +1,88 @@
+function negate(a) {
+  return !a;
+}
+
+function both(a, b) {
+  return a && b;
+}
+
+function either(a, b) {
+  return a || b;
+}
+
+function none(a, b) {
+  return !a && !b;
+}
+
+function one(a, b) {
+  return a !== b;
+}
+
+function truthiness(a) {
+  if (a) return true;
+  return false;
+}
+
+function isEqual(a, b) {
+  return a === b;
+}
+
+function isGreaterThan(a, b) {
+  return a > b;
+}
+
+function isLessThanOrEqualTo(a, b) {
+  return a <= b;
+}
+
+function isOdd(a) {
+  if (a % 2 !== 0) {
+    return true;
+  }
+  return false;
```

Comment:
You can update a lot of these functions to be more concise by simply returning the evaluation of the condition, for example here:
```
  return (a % 2 !== 0);
```
If it evaluates to `true`, you get `true` returned and vice versa